doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,118 @@
+# Install Kuadrant on a Kubernetes cluster
+
+> [!NOTE]  
+> You must perform these steps on each Kubernetes cluster where you want to use Kuadrant.
+
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, with `kubeadmin` or an account with similar permissions
+
+## Procedure
+
+If you are installing locally, consider running the kind & Kubernetes [quickstart script](https://docs.kuadrant.io/0.7.0/getting-started-single-cluster/).
```

Comment:
Not sure if it's an issue or worth pointing out, but should someone choose to setup their cluster via the quickstart script many of the steps afterwards in this guide are already handled such as kuadrant install, Gateway API install etc.

Reply:
Most of the steps outlined are perfectly fine to re-run but in the case of creating the aws-credentials secret you'll get an error stating that it already exists:

```
kubectl -n kuadrant-system create secret generic aws-credentials \
  --type=kuadrant.io/aws \
  --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
  --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
error: failed to create secret secrets "aws-credentials" already exists
```

Reply:
Yeah, `kubectl create`  operations are an explicit call to create a resource, whereas most of the other commands are `apply`'s - which have a different behaviour (create if not exists, update if matching, leave alone if the same). Could apply a secret too but then we'd have to inject YAML

Reply:
> Not sure if it's an issue or worth pointing out, but should someone choose to setup their cluster via the quickstart script many of the steps afterwards in this guide are already handled such as kuadrant install, Gateway API install etc.

You reckon we should be a bit more explicit to say this is an either or? e.g. If you're just looking to get setup, run quickstart and not this, otherwise, go through this guide. And don't do both (please).

Reply:
Yeah I think it would be good to state that certain steps are not required should you choose to follow the quick start process